ga4gh/datamodel/datasets.py

```diff
@@ -181,6 +213,12 @@ def __init__(
             seed = randomSeed + i
             variantSet = variants.SimulatedVariantSet(
                 self, localId, seed, numCalls, variantDensity)
+            cs = variantSet.getCallSets()
+            # Add biosamples
+            for c in cs:
```

Comment:
try and stay away from single/double letter variable names... we try and keep our names more descriptive and easy to read than that